docs/release-notes/8.14.asciidoc

```diff
@@ -26,6 +26,22 @@ On September 5, 2024, this issue was resolved.
 ====
 // end::known-issue-14686[]
 
+// tag::known-issue-192084[]
+[discrete]
+.Alerts wrongfully inherit previously selected tags
+[%collapsible]
+====
+*Details* +
+
+When you add tags to alerts in the Alerts table, the previously selected tags are incorrectly applied in addition to new ones that you select. 
```

Comment:
@dplumlee I figured the issue summary conveyed that the bug only exists in the UI. If it doesn't, I can change "in" to "from" to create a more direct relationship between the tagging functionality and the UI. WDYT?

```suggestion
When you add tags to alerts from the Alerts table, the previously selected tags are incorrectly applied in addition to the new ones that you select. 
```

Reply:
Yeah I think that's a simple fix!